package/scripts/aca/certificate_generate.sh

```diff
@@ -75,6 +75,10 @@ if [ $DOCKER_CONTAINER = true ]; then
     if [[ $(ss -t -l -n | grep -q LISTEN.*:::8009) -eq 0 ]]; then
         echo "Tomcat is running, so we restart it."
         /usr/libexec/tomcat/server stop
+        # Wait for Tomcat to stop completely and prevent port bind collisions
+        while [ -z $(tail -n 1 /var/log/tomcat/catalina.$(date +"%Y-%m-%d").log | grep "Destroying ProtocolHandler \[\"http-bio-8443\"\]") ]; do
```

Comment:
The argument passed to `-z` needs to be a single string or you will get an error. You need to wrap it in double quotes.
```suggestion
        while [ -z "$(tail -n 1 /var/log/tomcat/catalina.$(date +"%Y-%m-%d").log | grep "Destroying ProtocolHandler \[\"http-bio-8443\"\]")" ]; do
```